docs/src/components/BlockQuote/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from "./BlockQuote";
```

Comment:
Just for my education, why is this separate file with the export necessary?

Reply:
It's not, I assumed it's how we're writing other components, but I misunderstood our convention, I should've put all code inside of `BlockQuote/index.tsx`.